staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -74,8 +75,82 @@ type ValidatingAdmissionPolicy struct {
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	// Specification of the desired behavior of the ValidatingAdmissionPolicy.
 	Spec ValidatingAdmissionPolicySpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+	// Status is the status of the ValidatingAdmissionPolicy, including warnings that are useful to determine if the policy
```

Comment:
Let's use the serialized field names in godoc(`status`, `typeChecking`, `conditions`, etc.)

Reply:
Done. Pushed.